### PR TITLE
Handle Payments.

### PR DESCRIPTION
**Pull request by jopealgorta**

### CODE REVIEW

File: src/api/common/services/mercadopago.service.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export class MercadoPagoService {
         pending: `${getClientUrlOrNgrok(true)}/projects/${projectSlug}`,
       },
       auto_return: 'all',
-      notification_url: `${getSiteUrl()}/api/donations/webhook`,
+      // notification_url: `${getSiteUrl()}/api/donations/webhook`,
```
Comment (jstaricco10) [inline src/api/common/services/mercadopago.service.ts]: is this comment ok?

File: src/api/donations/entities/payment.entity.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { Field, Float, Int, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+class PaymentPayer {
+  @Field(() => String, { nullable: true })
```
Comment (jstaricco10) [inline src/api/donations/entities/payment.entity.ts:5]: is it ok that first and last name are nullable?

**Review by jstaricco10 — APPROVED** [review]

File: src/api/donations/entities/payment.entity.ts (hunk 2 above)
Comment (jopealgorta) [reply src/api/donations/entities/payment.entity.ts:5]: Yes, if the donor doesn't provide any.

File: src/api/common/services/mercadopago.service.ts (hunk 1 above)
Comment (jopealgorta) [reply src/api/common/services/mercadopago.service.ts]: Nope. Will change it.

**Pull request merged**